### PR TITLE
Add Ingrasys S8900 series platform

### PR DESCRIPTION
**Pull request by wadelnn**
* Add Ingrasys S8900-54XC platform driver
* Add Ingrasys S8900-64XC platform driver
* Reorganized Ingrasys debian package
* Move fancontrol config to device-specific file directory
* QSFP transceivers init on S9100-32X
* Update Thermal Policy in sensors.conf

Signed-off-by: Wade He <chihen.he@gmail.com>

### CODE REVIEW

File: device/ingrasys/x86_64-ingrasys_s8900_54xc-r0/INGRASYS-S8900-54XC/port_config.ini [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Ethernet44      81                Ethernet44
+Ethernet45      82                Ethernet45
+Ethernet46      83                Ethernet46
+Ethernet47      84                Ethernet47
+Ethernet48      85,86,87,88       Ethernet48
+Ethernet52      97,98,99,100      Ethernet52
+Ethernet56      101,102,103,104   Ethernet56
+Ethernet60      105,106,107,108   Ethernet60
+Ethernet64      109,110,111,112   Ethernet64
+Ethernet68      117,118,119,120   Ethernet68
```
Comment (jleveque) [inline device/ingrasys/x86_64-ingrasys_s8900_54xc-r0/INGRASYS-S8900-54XC/port_config.ini]: The `alias` column here is optional, and seeing as your aliases are exactly the same as their corresponding SONiC port names, the aliases serve no purpose for this device. You can safely remove the `alias` column in this file.

File: device/ingrasys/x86_64-ingrasys_s8900_64xc-r0/INGRASYS-S8900-64XC/port_config.ini [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+Ethernet38      40                 Ethernet38
+Ethernet39      37                 Ethernet39
+Ethernet40      41                 Ethernet40
+Ethernet41      42                 Ethernet41
+Ethernet42      43                 Ethernet42
+Ethernet43      44                 Ethernet43
+Ethernet44      48                 Ethernet44
+Ethernet45      45                 Ethernet45
+Ethernet46      46                 Ethernet46
+Ethernet47      47                 Ethernet47
```
Comment (jleveque) [inline device/ingrasys/x86_64-ingrasys_s8900_64xc-r0/INGRASYS-S8900-64XC/port_config.ini]: As with the file above, the `alias` column here is optional, and seeing as your aliases are exactly the same as their corresponding SONiC port names, the aliases serve no purpose for this device. You can safely remove the `alias` column in this file.

File: device/ingrasys/x86_64-ingrasys_s8900_54xc-r0/INGRASYS-S8900-54XC/port_config.ini (hunk 1 above)
Comment (wadelnn) [inline device/ingrasys/x86_64-ingrasys_s8900_54xc-r0/INGRASYS-S8900-54XC/port_config.ini]: Thanks for your recommendation. We will remove the alias column first and will name the specific alias later.

File: device/ingrasys/x86_64-ingrasys_s8900_64xc-r0/INGRASYS-S8900-64XC/port_config.ini (hunk 2 above)
Comment (wadelnn) [inline device/ingrasys/x86_64-ingrasys_s8900_64xc-r0/INGRASYS-S8900-64XC/port_config.ini]: Thanks for your recommendation. We will remove the alias column first and will name the specific alias later.

**Comment by lguohan** [comment]
:shipit:

**Pull request merged**